test/src/utils/react-dom-mock.js

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
+
+export function createPortal(element, container) {
```

Comment:
Hoping to learn what this does. Where is this used now? I only see the usage getting deleted.

Reply:
This file is used as an alias for `react-dom` during node tests, configured in ocularrc. Before we were hijacking the function with `require('react-dom').createPortal = createPortalMock`, but that no longer works in the ESM (import/export) world.

Reply:
Oh, ok got it. Could you comment that it's aliased in ocularrc?